src/index.js

```diff
@@ -6,6 +6,7 @@ import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
 import { ProductDataProvider } from "./context/product-data-context";
 import { AuthProvider } from "./context/auth-context";
+import { WishlistProvider } from "./context/wishlist-context";
```

Comment:
Can use barrel export so that all Providers can be imported directly from `./context`

Reply:
Thanks @znjs  done most of the barrel export and console error